### PR TITLE
Best practices grammar and spelling

### PR DESCRIPTION
**Pull request by waterseeker**
# I have made things!

## Checklist

- [x] I have double checked that there are no unrelated changes in this pull request (old patches, accidental config files, etc)
- [ ] I have created at least one test case for the changes I have made
- [ ] I have updated the documentation for the changes I have made
- [ ] I have added my changes to the `CHANGELOG.md`

## Related issues

🙏 Please, if you or your company is finding wemake-python-styleguide valuable, help us sustain the project by sponsoring it transparently on https://opencollective.com/wemake-python-styleguide. As a thank you, your profile/company logo will be added to our main README which receives hundreds of unique visitors per day.

### CODE REVIEW

File: wemake_python_styleguide/violations/base.py [hunk 1]
```diff
@@ -35,14 +35,14 @@
 Deprecating a violation
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-When you want to mark some violation as depracated,
+When you want to mark some violation as deprecated,
```
Comment (sobolevn) [inline wemake_python_styleguide/violations/base.py:38]: Looks like you have previous changes in your PR 😢

**Review by sobolevn — CHANGES_REQUESTED** [review]
Thanks! Can you please drop unrelated changes?
And then I will be happy to merge this PR.

**Comment by sobolevn** [comment]
Fixed it by myself. Thanks!

**Comment by waterseeker** [comment]
Thank you!

File: wemake_python_styleguide/violations/base.py (hunk 1 above)
Comment (waterseeker) [reply wemake_python_styleguide/violations/base.py:38]: My mistake, sorry about that.

**Pull request merged**